guides/v2.1/coding-standards/docblock-standard-general.md

```diff
@@ -539,39 +546,27 @@ Documentation is organized with the use of namespaces.
 ### @inheritdoc tag
 {:#inheritdoc}
 
-Whenever possible the `@inheritdoc` tag MUST be used for children to avoid duplication of doc blocks.
-
-Even Though PHPDocumentor understands inheritance and uses the parent doc block by default (without `@inheritdoc` tag specified), including the tag helps ensure that the doc block is not missed at all.
-
-Rules for usage of the tag:
-
-* Use `@inheritdoc` (notice no braces around) to indicate that the entire doc block should be inherited from the parent method.
-* Use the inline `{@inheritdoc}` tag (with braces around) in long descriptions to reuse the parent's long description. The tagged method MUST have its own short description.
+The `@inheritdoc` tag MUST NOT be used.
```

Comment:
This should be `SHOULD NOT` as the tag is supposed to be used for inline documentation. You may want to document the parent as _part_ of the child. Say for instance your child method just throws a bunch of exceptions. You'd use `@inheritdoc` and then `@throws` after it.

Reply:
Throwing new exception may be not the best example as it may break clients that depend on the parent. But adding long description that explains purpose of the method may require using `@inheritdoc`.

Reply:
done.